examples/react-client/fishjam-chat/src/components/RoomView.tsx

```diff
@@ -5,7 +5,7 @@ import { CallToolbar } from "./CallToolbar";
 import { nonNullablePredicate } from "@/lib/utils";
 
 export const RoomView = () => {
-  const { localPeer, remotePeers } = usePeers();
+  const { localPeer, remotePeers } = usePeers<{ displayName: string }>();
```

Comment:
**suggestion:** I think you already have `PeerMetadata` declared in
- `packages/react-client/src/types/internal.ts`

Peer metadata are also set locally so there must be at least one other place where the same type is used 🤔

Reply:
good catch because that type should be deleted